attentive-android-sdk/src/main/java/com/attentive/androidsdk/push/AttentivePush.kt

```diff
@@ -89,9 +88,10 @@ internal class AttentivePush {
         val body = remoteMessage.data.getOrElse("attentive_message_body") { null }
 
         if (title != null && body != null) {
-            //todo nullability check
-            val context = AttentiveEventTracker.instance.config?.applicationContext!!
-            sendNotification(title, body, remoteMessage.data, notificationIconId, context)
+            val context = AttentiveEventTracker.instance.config?.applicationContext
+            context?.let {
```

Comment:
added a null check just in case, instead of using !! to declare it will never be null.